src/main/java/me/totalfreedom/totalfreedommod/command/Command_ro.java

```diff
@@ -37,22 +37,32 @@ public static int replaceBlocks(Location center, Material fromMaterial, Material
                     BlockData data = block.getBlockData();
                     if (block.getLocation().distanceSquared(center) < (radius * radius))
                     {
-                        if (fromMaterial.equals(Material.WATER) && data instanceof Waterlogged)
+                        if (material.equals(Material.WATER) && data instanceof Waterlogged waterloggedData)
                         {
-                            Waterlogged waterloggedData = (Waterlogged)data;
+                            if (plugin.cpb.isEnabled())
+                            {
+                                plugin.cpb.getCoreProtectAPI().logRemoval(user, block.getLocation(), material, data);
```

Comment:
Since we're using `getCoreProtectAPI()` more than once, can we consider making it a variable so we don't have to call the get method more than once?

Reply:
Done